MonoGame.Framework/Graphics/Texture2D.OpenGL.cs

```diff
@@ -70,73 +70,61 @@ public partial class Texture2D : Texture
         private void PlatformConstruct(int width, int height, bool mipmap, SurfaceFormat format, SurfaceType type, bool shared)
         {
             this.glTarget = TextureTarget.Texture2D;
+            format.GetGLFormat(GraphicsDevice, out glInternalFormat, out glFormat, out glType);
+            // GL texture is generated on first access in PlatformSetData
```

Comment:
I'm unsure about this.  On DirectX after this method the texture fully exists and can be used to render with.  If you rendered with a setting to force rendering with a mipmap beyond the top level... it would work.  If you ran any DirectX debugger tool... you would see a valid constructed texture with a full mip chain.

Can we say the same about this for OpenGL?